README.rst

```diff
@@ -73,6 +73,11 @@ Requirements
 
 Parsl is supported in Python 3.5+. Requirements can be found `here <requirements.txt>`_. Requirements for running tests can be found `here <test-requirements.txt>`_.
 
+Code of Conduct
+============
+
+Parsl seeks to foster an open and welcoming environment - Please see the <a href="https://github.com/Parsl/parsl/blob/master/CoC.md">Parsl Code of Conduct</a> for more details.
```

Comment:
I think <a syntax doesn't work: see what it looks like by clicking View File on one of the top right drop down menus (https://github.com/Parsl/parsl/blob/2b161a66b5cb5d6b77bf18bcf3e718afefaf2917/README.rst for this commit)

Reply:
I think I've fixed this